app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/Sub02Controller.scala

```diff
@@ -110,8 +110,6 @@ class Sub02Controller @Inject() (
         processedDate <- sessionCache.sub01Outcome.map(
           sub01 => if (sub01.processedDate.nonEmpty) sub01.processedDate else sub02Outcome.processedDate
         )
-        _ <- sessionCache.remove
```

Comment:
Did we not decide that we need this? So that the cache was empty of everything other than the sub02Outcome?

Reply:
Yeah we did, I forgot I had made this change in reg as well, will change it back